frontend/src/functions/getPlan.ts

```diff
@@ -15,9 +17,15 @@ const getPlan = <P extends Partial<PlanData>>(
 
 export const getPeriodicalPremiumPrice = (
   runtimeData: RuntimeDataWithPeriodicalPremiumAvailable,
-  billingPeriod: keyof PlanData
+  billingPeriod: keyof PlanData,
+  l10n: ReactLocalization
```

Comment:
suggestion (non-blocking): can we save ourselves this extra parameter & argument on all the calls if we

`import { ReactLocalization, useLocalization } from "@fluent/react";`

and then

`const { l10n } = useLocalization();`

in these functions?

Reply:
[@lloan had exactly the same question](https://github.com/mozilla/fx-private-relay/pull/2507#discussion_r971484739) :)